Website/src/rendering/src/components/PreviewSearch/LeftColumn.tsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+const PreviewSearchList = ({
+  items = [],
+  title = '',
+  onMouseEnter = (text: string) => {
+    console.log(text);
+  },
+  onMouseLeave = () => {},
+  redirectUrl = '',
+}): JSX.Element => {
+  return window.RFK.ui.html` <div class="list-wrapper">
+    ${
+      items.length > 0 &&
+      window.RFK.ui.html` <h2 class="list-wrapper__title">${title}</h2>
+      <ul>
+        ${items.map(
+          ({ text, id, url }) => window.RFK.ui.html` <li
+            class="list-wrapper__item"
+            id=${id}
+            onMouseEnter=${() => onMouseEnter(text)}
+            onMouseLeave=${onMouseLeave}
+          >
+            ${
+              url
+                ? window.RFK.ui.html`<a href="${url}">${text}</a>`
+                : window.RFK.ui.html`<a href="${redirectUrl}${text}">${text}</a>`
+            }
+          </li>`
+        )}
+      </ul>`
+    }
+  </div>`;
+};
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
I see many eslint disable comments in this PR. Why are the reasons they are needed? Can the code be reworked to remove the comments?

Reply:
I think I see what's going on here now after investigating further. The examples from the RFK Preview Search Widget integration I was following used a lot of object destructuring, but I need to instead create types for props and pass those in instead. Will work on making those changes now.